client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -187,7 +188,10 @@ const ManageSelectedSiteSidebar = ( { path }: { path: string } ) => {
 						? {
 								label: translate( 'Site Settings' ),
 								icon: chevronLeft,
-								onClick: () => redirectPage( '/dashboard' ),
+								onClick: () => {
+									recordTracksEvent( 'calypso_jetpack_sidebar_site_settings_back_button_click' );
```

Comment:
I feel relatively certain on this, but best to double-check: I believe in Calypso, it's recommended to use `calypso/state/analytics/actions` instead of `@automattic/calypso-analytics` for recording Tracks Events, because of the way it affects the Redux state tree. I could be wrong, but I thought it important enough to bring up, at least.